.claude/agent-memory/game-archaeology-specialist/world-entry-resolved.md

```
@@ -5,12 +5,22 @@ metadata:
   type: project
 ---
 
+> [!WARNING] Confidence: STALE — Q1 contradicted by V5 W-enable-entities finding
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)